pulsar-io/aerospike/pom.xml

```diff
@@ -36,16 +36,19 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>pulsar-io-core</artifactId>
       <version>${project.version}</version>
+      <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
+      <scope>provided</scope>
```

Comment:
Why do we even need these dependencies here (other than `pulsar-io-core`)? 
Is the connector making direct use of Jackson? or is this used for tests?

Reply:
Yes, there's direct usage of Jackson in https://github.com/apache/pulsar/blob/65fed8b7fc258a2980b522f32db0f37f017bf84d/pulsar-io/aerospike/src/main/java/org/apache/pulsar/io/aerospike/AerospikeSinkConfig.java#L22-L23

Reply:
Gotcha. As a side note, we could provide here a standard method to perform the yaml ser/de since most (all?) connectors are going to do this. 

This way, the connector don't need to depend on that and the serialization code is already embedded in the runtime.